### ISSUE TITLE
Console errors in production

### ISSUE THREAD

**Issue by art-mironoff** [RakanNimer/react-google-charts#89]
In production, sometimes I have errors that occur in this piece of code.

```
componentWillUnmount() {
    try {
        google.visualization.events.removeAllListeners(this.wrapper);
    }
    catch(err) {
        console.error("Error removing events, error : ", err);
    }
}
```
They appear because the component on my page rerender several times. Does anyone faced with this problem and how to fixed it. Chart works well, just annoying errors in the console.

**Comment by paolorovella** [comment]
I've the same problem 😕

**Comment by RakanNimer** [comment]
Fixed in 1.4.2 !
